src/main/java/hudson/plugins/im/bot/SetAliasCommand.java

```diff
@@ -89,7 +89,7 @@ public String getHelp() {
         public AliasCommand(BotCommand cmd, String commandName, String[] arguments) {
 			this.command = cmd;
 			this.commandName = commandName;
-			this.arguments = arguments;
+			this.arguments = Arrays.copyOf(arguments, arguments.length);
```

Comment:
Exposure of a mutable array caused a find bugs warning here (many other changes in this PR relate to find bugs)